File: PHP Excel Examples/GeneralApiDemo/convertInRequestToPath.php
<?php

use Spire\Cloud\Excel\Sdk\Api\GeneralApi;
use Spire\Cloud\Excel\Sdk\Configuration;
use Spire\Cloud\Excel\Sdk\Model\ExportFormat;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new GeneralApi($configuration);

$document = "D:/input/charts.xlsx";
$password = null;
$outPath ="output/ConvertInRequestToPath.xlsx";
$format = ExportFormat::XLSX; //Supported formats : Xlsx, Xlsb, Xls, Ods, Pdf, Xps, Ps, Pcl
$apiInstance->convertInRequestToPath($format, $outPath, $document, $password);

File: Python Excel Examples/GeneralApiDemo/convertInRequestToPath.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.general_api import GeneralApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.general_api.GeneralApi(configuration)
format = "Pdf"   #Supported formats: Xlsx/Xls/Xlsb/Ods/Pdf/Xps/Ps/Pcl
file = "D:/inputFile/charts.xlsx"
password = ""
destPath = "/output/charts.pdf"
api.convert_in_request_to_path(format, destPath,file=file, password=password)